src/Cedar/Proto_PPP.c

```diff
@@ -1278,6 +1278,7 @@ bool PPPProcessEAPResponsePacket(PPP_SESSION *p, PPP_PACKET *pp, PPP_PACKET *req
 		switch (eap_packet->Type)
 		{
 		case PPP_EAP_TYPE_IDENTITY:
+			p->Eap_MatchUserByCert = false;

```

Comment:
The fix is correct but the logic is hard to follow. We can just give `authtype` a initial value.

Reply:
I considered doing that, but I couldn't select one tbh. I'd leave this and add a default value, just which one?

Reply:
Reason why I'd leave it - if the client for some weird reason resends an identity packet, better to reset it here anyway.